code/game/objects/items/fireaxe.dm

```diff
@@ -74,3 +74,18 @@
 
 /obj/item/fireaxe/boneaxe/update_icon_state()
 	icon_state = "bone_axe0"
+
+/*
+ * Metal Hydrogen Axe
+ */
+/obj/item/fireaxe/metal_h2_axe  // Blatant imitation of the fireaxe, but made out of metallic hydrogen
+	icon_state = "metalh2_axe0"
+	name = "metallic hydrogen axe"
+	desc = "A large, menacing axe made of an unkown substance that the most elder atmosians calls Metallic Hydrogen. Truly an otherwordly weapon."
+
+/obj/item/fireaxe/metal_h2_axe/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_unwielded=5, force_wielded=23, icon_wielded="metalh2_axe1")
```

Comment:
i think that 24 would be fine, thats what the original fire axe has and 1 point of force doesnt make much difference

Reply:
Believe it or not there's a precedent here, albeit in the other direction. The plasmaglass spear gets 1 extra point of force of all types. It doesn't make an actual difference in reality, because you still need 6 hits to crit, but it's a little bit of flavor basically.

Reply:
i'll keep this one at 23 just to make it clear that is not the "original" fireaxe